packages/skill-api/src/core/pages/index.ts

```diff
@@ -0,0 +1,42 @@
+import renderToString from 'preact-render-to-string'
+import TestPage from './test'
+import {InternalOptions} from '../types'
+import {IncomingRequest, OutgoingResponse} from '../index'
+
+type RenderPageParams = {
+  query?: IncomingRequest['query']
+  cookies?: any[]
+} & Partial<Pick<InternalOptions, 'url' | 'theme'>>
+
+function css() {
+  return ''
+}
+
+export default function renderPage(params: RenderPageParams) {
```

Comment:
pretty cool way to build consistent internal pages with a theme and consistent configurable styling